readme.md

```diff
@@ -1,56 +1,46 @@
 ### fastapi for to serve geodata
 
 
-https://fastapi.tiangolo.com/advanced/templates/
-https://pybit.es/articles/how-to-handle-environment-variables-in-python/
+Access postges db with Python Fastapi backend and render map using Leaflet.js. 
 
+### Table of Contents
 
+    Installation
+    Usage
+    API Documentation
+    License
 
+### Installation
 
-Access postges db with Python Fastapi backend and Leaflet.js. Use HTMX to minimize javascript complexity.
+```bash
+git clone https://github.com/thisAKcode/fastapi_streetart_map.git
+pip install -r requirements.txt
+cd app
+python dbpg2.py
+uvicorn main:app --reload
 
```

Comment:
navigate to http://127.0.0.1:8000/map/

Reply:
> Nice, tried it, works (apart from some missing packages I had to install), any code to review?
since it is not a pr that properly done  I can't reference where I am stuck. Let's look at it during coding session?

Reply:
ok